src/tracks/playabletrack/wavetrack/ui/SpectrumCache.h

```diff
@@ -39,36 +39,35 @@ class AUDACITY_DLL_API SpecCache {
    {
    }
 
-   bool Matches(int dirty_, double pixelsPerSecond,
-      const SpectrogramSettings &settings, double rate) const;
+   bool Matches(
+      int dirty_, double samplesPerPixel,
+      const SpectrogramSettings& settings) const;
 
    // Calculate one column of the spectrum
-   bool CalculateOneSpectrum
-      (const SpectrogramSettings &settings,
-       const WideSampleSequence &sequence,
-       const int xx, sampleCount numSamples,
-       double offset, double rate, double pixelsPerSecond,
-       int lowerBoundX, int upperBoundX,
-       const std::vector<float> &gainFactors,
-       float* __restrict scratch,
-       float* __restrict out) const;
+   // This assumes that all clips have equal stretch ratio, i.e., no stretch
+   // handle is used, and all clips were added at a given project tempo.
```

Comment:
Is this comment in commit "Spectrum Display..." correct?

Reply:
It's outdated, stemming from the time when spectrum display did not account for stretching. Thanks.